### ISSUE TITLE
Reader appearance controls are confusing to use non-visually

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2251]
## Observed Behavior
Each of the appearance controls have issues, I have grouped them into one issue because I think the whole thing could benefit from a quick redesign. To assistive technologies these complex groupings of controls appear to be one long list of buttons.

## Expected Behavior
It is expected that each of these groupings of controls be properly labeled as discussed in issue #2250.
### Font style and size
Essentially this control is a toggle and two related buttons. This should be implemented as two radio buttons and two regular buttons. Additionally the font size controls should be focus able with the keyboard even when they are disabled. This allows assistive technology users to quickly understand the functionality and state of the interface. If the current layout of having size controls for both font types is to be kept, the size controls for the font type not currently in use should be disabled.
### Font Grouping
This is essenially a toggle. Should be implemented as two radio buttons.
### Text Margin
This one ... well actually this one is fine I guess just make sure that the buttons can be focused even when they are disabled. 

---
A sketch of what this all of this might look like from a screen reader users perspective: (in this example the user has maximum font size and minimum margin size)
```
List of three items
  Font grouping
    Serif radio button, not checked, 1 of 2
    Sans-Serif radio button, checked, 2 of 2
    Decrease font size button, disabled
    Increase font size button
  Color scheme grouping
    Light color scheme radio button, checked, 1 of 2
    Dark color scheme radio button, no checked 2 of 2
  Text margin grouping
    increase text margin size button
    decrease text margin size button, disabled
Close panel button
```
## Test Environment
NVDA 2019.1.1
ESR Firefox 60.7.0esr

**Comment by avaaec** [comment]
Fix verified. Looks great!

**Comment by zdavis** [comment]
Related: #2320
